src/app/g3w-ol/controls/streetviewcontrol.js

```diff
@@ -39,18 +39,30 @@ const StreetViewControl = function(options={}) {
   this._panorama = null;
   this._map = null;
   this._projection = null;
-  this._lastposition = null;
+  /**
+   * Object contain previuos data referred to :
+   * - rotation: for array rotation
+   * - resolution: map view resolution
+   * - position: lat lng of streetview
+   * @type {{rotation: null, position: null, resolution: null}}
+   */
+  this.cached = {
+    position: [0, 0],
+    resolution: null,
+    rotation: null
+  };
```

Comment:
I made some tests, but even so the position seems to me to be often wrong (ie. not so deterministic)